sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,28 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
     packages = ('hbn-repo',)
 
     def setup(self):
         self.add_copy_spec([
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/containers',
+            '/var/log/sfc-install.log',
+            '/tmp/sf_devices',
+            '/tmp/sfr_devices',
+            '/tmp/sfc-activated',
```

Comment:
do we want also the tmp files used for env variables?

Reply:
I will try to add those.